vimrc

```diff
@@ -75,7 +75,6 @@ if executable('ag')
 endif
 
 " Color scheme
-colorscheme github
 highlight NonText guibg=#060606
```

Comment:
I think this is for highlighting extra whitespace, which I think is good for any colorscheme. Reference https://wincent.com/blog/making-vim-highlight-suspicious-characters

Thoughts?